extension/src/experiments/index.ts

```diff
@@ -529,7 +529,9 @@ export class Experiments extends BaseRepository<TableData> {
       return
     }
 
-    const paramsToModify = await this.pickAndModifyParams()
+    const paramsToModify = await this.pickAndModifyParams(
+      'Use range(start,stop,step) or choice(*choices) to add multiple experiments to the queue'
```

Comment:
[F] Trying to be explicit and verbose.

`*choices` is Python syntax for `...choices`